### PR TITLE
Adapted docummentation for DAVIS 2019 Challenge

### PR DESCRIPTION
**Pull request by scaelles**

### CODE REVIEW

File: docs/pages/challenge.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+The interactive track is built on the DAVIS 2017 dataset. The video sequences in the `train`, `val` and `test-dev` subsets have been manually annotated with scribbles. The annotators were instructed to label all objects of a sequence in a representative frame (not necessarily the first frame of the sequence as in the semi-supervised track).
 
-For every sequence of DAVIS 2017 there are 3 different sets of scribbles, annotated by different users. The figure below illustrates an example of three different user annotations on the same sequence `dogs-jump`. Note that annotations were performed on different frames, chosen by the respective users.
+For every sequence in the `train`, `val` and `test-dev` subsets of DAVIS 2017, there are 3 different sets of scribbles annotated by different users. The figure below illustrates an example of three different user annotations on the same sequence `dogs-jump`. Note that annotations were performed on different frames, chosen by the respective users. More information on how to download the scribbles in the [Installation](/user_guide/installation) guide.
```
Comment (albertomontesg) [inline docs/pages/challenge.md]: user_guide/installation.md

File: docs/pages/challenge.md [hunk 2]
```diff
@@ -45,27 +45,27 @@ This framework also provides the possibility to evaluate the methods locally. Lo
...
+In order to submit results to the interactive challenge, this framework allows to evaluate models agains a remote server. For remote evaluation, only the `test-dev` subset is available and the results are used for the ranking of the challenge.
 
-In order to participate to the challenge a registration is required. To register, please go to https://server.davischallenge.org and fill in the form with your information. A mail will be sent to the provided email with a user key required for the remote evaluation. This key should be put in your code in order to identify every user.
+In order to participate to the challenge a registration is required. To register, please go to https://server.davischallenge.org and fill in the form with your information. A mail will be sent to the provided email with a user key required for the remote evaluation. This key should be introduced in your code in order to identify your submissions. See the [Usage](/user_guide/usage) guide for more details.
```
Comment (albertomontesg) [inline docs/pages/challenge.md]: Same notation here. Use relative links

File: docs/pages/challenge.md [hunk 3]
```diff
@@ -45,27 +45,27 @@ This framework also provides the possibility to evaluate the methods locally. Lo
...
 
-In the end, a curve showing Jaccard as a function of Accumulated Time will be generated. In the following example you can see an example of how the curve looks like for a baseline method [Scribble-OSVOS](https://github.com/kmaninis/Scribble-OSVOS):
+At the end, a curve showing Jaccard as a function of Accumulated Time is generated. In the following example you can see an example of how the curve looks like for a baseline method [Scribble-OSVOS](https://github.com/kmaninis/Scribble-OSVOS):
```
Comment (albertomontesg) [inline docs/pages/challenge.md]: We will plot J&F no?

File: docs/pages/user_guide/usage.md [hunk 4]
```diff
@@ -81,16 +81,16 @@ with DavisInteractiveSession(host='localhost', davis_root='path/to/davis') as se
...
 * [`scribbles2points`](/docs/utils.scribbles): from the scribble, it extracts all the (x, y) coordinates of all the line points as well as its object ID.
 
 If you think there is any new transformation or a modification to the current ones that might be useful to work with scribble data, please do not hesitate to send a [pull request](https://github.com/albertomontesg/davis-interactive/pulls).
 
 ## Prediction Submission
 
-After each iteration, it is mandatory to submit the scribbles to evaluate and at the same time to automatically generate the scribble for the next iteration.
+At the end of each iteration, the user must submit the mask predicted by his/her model to be evaluated in the server. As an optional parameter, the user may specify which frames have to be considered in order to compute the next scribbles (using the paramter `next_scribble_frame_candidates` in the `submit_masks` function). By default, all the frames in a sequences are considered.
```
Comment (albertomontesg) [inline docs/pages/user_guide/usage.md]: parameter

File: docs/pages/challenge.md (hunk 3 above)
Comment (scaelles) [reply docs/pages/challenge.md]: Fix in the next commit

File: docs/pages/challenge.md (hunk 1 above)
Comment (scaelles) [reply docs/pages/challenge.md]: Fixed in latest commit

File: docs/pages/challenge.md (hunk 2 above)
Comment (scaelles) [reply docs/pages/challenge.md]: Fixed in latest commit

File: docs/pages/user_guide/usage.md (hunk 4 above)
Comment (scaelles) [reply docs/pages/user_guide/usage.md]: Fixed in latest commit

**Pull request merged**